netstandard/ref/System.IO.Compression.cs

```diff
@@ -27,16 +27,18 @@ public partial class DeflateStream : System.IO.Stream
         public override bool CanWrite { get { throw null; } }
         public override long Length { get { throw null; } }
         public override long Position { get { throw null; } set { } }
-        public override System.IAsyncResult BeginRead(byte[] array, int offset, int count, System.AsyncCallback asyncCallback, object asyncState) { throw null; }
+        public override System.IAsyncResult BeginRead(byte[] buffer, int offset, int count, System.AsyncCallback asyncCallback, object asyncState) { throw null; }
```

Comment:
This is a breaking change, very minor, but still.

Reply:
This is not breaking change as C# compiler (which NS assumes) ignores override names

Reply:
It is a source-breaking change which according to https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md#signatures is disallowed.

@terrajobst does the same policy apply for .NET Standard too?

Reply:
The coding guideline is inaccurate as should not apply to overrides. It's breaking change only for static, virtual and abstract members with parameters.

Reply:
@marek-safar why do you think the renamed parameters isn't a breaking change for overrides? 

While it is technically a breaking change it is only a source break and we have been pretty loose in enforcing it because it is only an issue if folks use named parameters in source. I think in general we allow for consistency across platforms to override this breaking change.

Reply:
So we're in agreement to leave this as proposed here then?